yugabyte/src/yugabyte/auto.clj

```diff
@@ -413,7 +421,8 @@
             :--enable_tracing
             :--rpc_slow_query_threshold_ms 1000
             (tserver-api-opts (:api test) node)
-            (tserver-workload-specific-opts test)
+            (tserver-read-committed-flags test)
+            (master-tserver-pessimistic-locking-flags test)
```

Comment:
are we still running it /without/ wait on conflict?

also can we rename this `master-tserver-wait-on-conflict-flags`?